tools/deploy.sh

```diff
@@ -67,7 +67,7 @@ function deploy() {
         do
             echo "$SCHEMA"
             rm -f tools/db-dumps/ibf_$SCHEMA.dump
-            PGPASSWORD=$DB_PASSWORD pg_dump -U $DB_USERNAME -Fc -f tools/db-dumps/ibf_$SCHEMA.dump -h $DB_HOST -n \"$SCHEMA\" $DB_DATABASE
```

Comment:
@gulfaraz I changed this back. The point here is that it is dumping 
- from **geonode_datav3**
- to **$DB_DATABASE** (that being 'test' or 'staging' depending on which server the script is run on)

Reply:
so **geonode_datav3** is like a temp database?

Reply:
@gulfaraz no, it's the development database